tsconfig.test.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
What about the update to eslint-config made having a separate tsconfig for tests necessary?